scheduler/reconcile_test.go

```diff
@@ -5272,16 +5272,56 @@ func TestReconciler_Node_Disconnect_Updates_Alloc_To_Unknown(t *testing.T) {
 	})
 }
 
+func TestReconciler_Disconnect_UpdateJobAfterReconnect(t *testing.T) {
+	ci.Parallel(t)
+
+	// Create 2 allocs and simulate one have being previously disconnected.
```

Comment:
```suggestion
	// Create 2 allocs and simulate one have being previously disconnected and then reconnected.
```

I think, based on these alloc states?

Reply:
Oh that's right, good catch!